clean_ceph_vols.py

```diff
@@ -91,8 +87,9 @@ def cleanup_ceph_vols(osp_cred):
     with parallel() as p:
         for volume in driver.list_volumes():
             if volume.state in vol_states:
-                p.spawn(volume_cleanup, volume, driver)
-                sleep(1)
+                if volume.name[:5] in "ceph-":
```

Comment:
IMO, we could do `if volume.name.startswith('ceph-'):`

Reply:
modified the search cmd